tools/python/check_model_can_use_ort_mobile_pkg.py

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Helper script that will check if the types and operators used in an ONNX model
+# would be supported by the pre-built ORT Mobile package.
+
+import argparse
+import onnx
+import os
+import pathlib
+import sys
+from util import reduced_build_config_parser
+
+
+def check_graph(graph, opsets, required_ops, global_types, unsupported_ops):
```

Comment:
so we assume that:
- globally allowed types are specified
- only model inputs need to be checked and not intermediate values

is that generally obvious or would it be worth documenting?

Reply:
We use the global type list in the pre-built package config so I think it's fair to assume that. 

Checking more things gets much harder. As it's the ONNX model there's no type info for intermediate values in the model. Additionally it gets pretty nuanced with things like the int32/int64 support being a required type on specific inputs/outputs of specific operators. I didn't want to re-create all that logic at this point as I'm not sure how frequently a model not being able to be run in the pre-built package is due to the inner types used. Mainly trying to have an ability to quickly assess an ONNX model (e.g. we just exported from PT) being likely to be able to run with the pre-built package. 

I'll see if there are some other simple checks to add without going overboard.

Reply:
I added a few more checks and tested against all the models used in the generation of the pre-built package config. With the change there are no more false negatives, and the messaging calls out that it's not a complete check.